components/messenger.rst

```diff
@@ -86,6 +86,65 @@ that will do the required processing for your message::
        }
     }
 
+Envelope
```

Comment:
The title of this section and the introduction is not very clear to me. Here is a reword proposal:

Before:

```rst
Envelope
--------

The notion of an envelope is a concept that helps add context around the
messages. An envelope is a message and a set of data. From a user's perspective, this
allows you to set some configuration around the message. For example, to set the serialization
groups used when the message goes through the transport layer, wrap your message
in an ``Envelope`` and add some ``SerializerConfiguration``::
```

After:

```rst
Adding Metadata to Messages (Envelopes)
---------------------------------------

If you need to add metadata or some configuration to a message, wrap it with the
:class:`Symfony\\Component\\Messenger\\Envelope` class. For example, to set the
serialization groups used when the message goes through the transport layer, use
the ``SerializerConfiguration`` envelope::
```

Reply:
Much better 👍